unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -36,26 +34,65 @@ public class FriendsController : IFriendsController
         public event Action<FriendRequest> OnFriendRequestReceived;
         public event Action<FriendRequest> OnSentFriendRequestApproved;
 
-        public static void CreateSharedInstance(IFriendsApiBridge apiBridge)
-        {
-            i = new FriendsController(apiBridge);
-        }
+        private CancellationTokenSource controllerCancellationTokenSource;
 
-        public FriendsController(IFriendsApiBridge apiBridge)
+        public FriendsController(IFriendsApiBridge apiBridge, IRPCSocialApiBridge rpcSocialApiBridge)
```

Comment:
i think it should be able to merge into the same `IFriendsApiBridge` so you inject a single bridge

Reply:
The APIs will diverge too much for them to be the same bridge, it makes no sense to inject only one bridge, one is not the fallback of the other